### DISCUSSION TITLE
How do stop this from happening?

### DISCUSSION THREAD

**Discussion by HarizRe** [YimMenu/YimMenu#3445] [Q&A]
![image](https://github.com/user-attachments/assets/dbc56124-a8cd-401a-b9fc-a6bca9f5bd0e)

Everytime i play mission and there is a vehicle that i spawned, the vehicle is still exist even when i already die and the mission already restarted..

**Comment by lonelybud** [comment]
related https://github.com/YimMenu/YimMenu/pull/3434

**Comment by maybegreat48** [comment]
Fixed by #3434
